packages/test/local-server-stress-tests/src/stressDataObject.ts

```diff
@@ -0,0 +1,320 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { stringToBuffer } from "@fluid-internal/client-utils";
+import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct/internal";
+import {
+	AttachState,
+	type IRuntimeFactory,
+} from "@fluidframework/container-definitions/internal";
+import {
+	loadContainerRuntime,
+	RuntimeHeaders,
+	type IContainerRuntimeOptionsInternal,
+} from "@fluidframework/container-runtime/internal";
+// eslint-disable-next-line import/no-deprecated
+import type { IContainerRuntimeWithResolveHandle_Deprecated } from "@fluidframework/container-runtime-definitions/internal";
+import type {
+	IFluidHandle,
+	FluidObject,
+	IFluidLoadable,
+} from "@fluidframework/core-interfaces";
+import { assert, Lazy, LazyPromise } from "@fluidframework/core-utils/internal";
+import type { IChannel } from "@fluidframework/datastore-definitions/internal";
+import { ISharedMap, SharedMap } from "@fluidframework/map/internal";
+import { toFluidHandleInternal } from "@fluidframework/runtime-utils/internal";
+
+import { ddsModelMap } from "./ddsModels";
+import { makeUnreachableCodePathProxy } from "./localServerStressHarness";
+
+export interface UploadBlob {
+	type: "uploadBlob";
+	tag: `blob-${number}`;
+}
+export interface CreateDataStore {
+	type: "createDataStore";
+	asChild: boolean;
+	tag: `datastore-${number}`;
+}
+
+export interface CreateChannel {
+	type: "createChannel";
+	channelType: string;
+	tag: `channel-${number}`;
+}
+
+export type StressDataObjectOperations = UploadBlob | CreateDataStore | CreateChannel;
+
+export class StressDataObject extends DataObject {
+	public static readonly factory = new Lazy(() => {
```

Comment:
The Lazy/LazyPromise here feels like overcomplicating.

Reply:
its necessary to avoid a circular dependency in instantiation, as the factory takes the class

Reply:
Ugh aqueduct.  Still should be able to drop the Lazy though.  Either:

```ts
// top level
const factory = new DataObjectFactory(
	"StressDataObject",
	StressDataObject,
	[...ddsModelMap.values()].map((v) => v.factory),
	{},
	[["StressDataObject", new LazyPromise(() => factory)]],
);

// ...

public static readonly factory = factory;
```

or

```ts
public static readonly factory = new DataObjectFactory(
	"StressDataObject",
	StressDataObject,
	[...ddsModelMap.values()].map((v) => v.factory),
	{},
	[["StressDataObject", new LazyPromise(() => StressDataObject.factory)]],
);
```